tests/test_encrypt.py

```diff
@@ -263,7 +263,7 @@ def test_encrypt_simple_contest_referendum_succeeds(self):
         self.assertTrue(result.is_valid_encryption(hash_context, keypair.public_key))
 
     @settings(
-        deadline=timedelta(milliseconds=2000),
+        deadline=timedelta(milliseconds=4000),
```

Comment:
Do these still go up in time?  Shouldn't these stay the same with the improvements?

Reply:
The Mac VM seems to have some variance on the amount of time it takes to run these tests.  i was getting false failures due to the timeout being too short so i increased them.